config/default.json

```diff
@@ -47,14 +47,19 @@
       "enabled": "true"
     },
     "allowSaveAndReturn": {
-      "enabled": "false"
+      "enabled": false
```

Comment:
doesn't matter really because the config module cannot read booleans. It always transforms into Strings.